### PR TITLE
Build.ConfigHeader: render identifiers

### PR DESCRIPTION
**Pull request by alichraghi**

### CODE REVIEW

File: lib/std/Build/Step/ConfigHeader.zig [hunk 1]
```diff
@@ -553,6 +553,12 @@ fn replace_variables(
                     allocator.free(content_buf);
                     content_buf = buf;
                 },
+                .ident => |ident| {
```
Comment (andrewrk) [inline lib/std/Build/Step/ConfigHeader.zig:556]: This looks like the same body as `string`. Couldn't you just do `.string, .ident => |x| {`

**Review by andrewrk — CHANGES_REQUESTED** [review]
Please either provide a test, or an explanation for what this change does and how you tested it.

**Comment by alichraghi** [comment]
while adding [`build.zig`](https://github.com/hexops/ogg/blob/b6cc26bd14a5f99f06359f39772efd6e8c90aa3f/build.zig#L16-L21) to libogg, i noticed that this [config header](https://github.com/xiph/ogg/blob/db5c7a49ce7ebda47b15b78471e78fb7f2483e22/include/ogg/config_types.h.in#L19-L24) requires identifier values.
before this PR, identifiers were ignored and rendered as `typedef  ogg_int16_t;` instead of `typedef int16_t ogg_int16_t;`.

**Comment by andrewrk** [comment]
btw https://github.com/andrewrk/libogg

**Comment by alichraghi** [comment]
yeah i see that one but somehow opusfile requires `config_types.h`

**Pull request merged**